Lombiq.Tests.UI/Services/WebDriverFactory.cs

```diff
@@ -50,6 +50,9 @@ ChromeDriver CreateDriverInner(ChromeDriverService service)
             chromeConfig.Options.AddArgument("force-device-scale-factor=1");
             chromeConfig.Options.AddArgument("high-dpi-support=1");
 
+            // Disabling smooth scrolling
+            chromeConfig.Options.AddArgument("disable-smooth-scrolling");
```

Comment:
Can this be done for the other browsers too?

Reply:
Edge: Probably, it's chromium based.
Firefox: I can't find appropriate docs, but maybe.
IE: I think, no.

Reply:
It seems all tests pass with Edge, Chrome, and Firefox on both GH windows-latest and ubuntu-latest. On TC, the tests pass with Edge and Chrome, but not with Firefox (because it's not installed based on the exception message).

Reply:
Awesome! Yep, and no need to further test on TC.

Reply:
Ok, then I'll remove Firefox annotation from tests, and what about Edge and related code changes, keep it, or drop it?

Reply:
Keep everything apart from the `Firefox` attribute on the test